doc/rados/operations/erasure-code.rst

```diff
@@ -1,14 +1,14 @@
 .. _ecpool:
 
-=============
+==============
  Erasure code
-=============
+==============
 
 By default, Ceph `pools <../pools>`_ are created with the type "replicated". In
-replicated-type pools, every object is copied to multiple disks (this
-multiple copying is the "replication").
+replicated-type pools, every object is copied to multiple disks. This
```

Comment:
^ double quotes -> backticks

s/-type//
s/every/each/

V The last sentence, former parenthetical, is IMHO redundant and bears removal.

Reply:
By itself, it seems redundant, but we need it because it sets up the following paragraph.